src/components/Checkbox/Checkbox.module.scss

```
@@ -0,0 +1,137 @@
+@import '../../styles/colors';
+
+button {
+  all: unset;
+}
+
+.Container {
+  display: flex;
+  align-items: center;
+  padding: 4px;
+}
+
+.Container:focus-within {
+  border-radius: 8px;
+  outline: 1px solid #{$green-11};
+}
+
+.Container:hover {
+  cursor: pointer;
+
+  .Label:hover {
+    cursor: pointer;
+  }
+}
+
+.CheckboxRoot {
+  min-width: 14px;
+  min-height: 14px;
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #{$green-11};
+
+  .CheckboxIndicator {
+    color: #{$dark-secondary};
+  }
+}
+
+.CheckboxRoot:hover {
+  background-color: transparentize($green-11, 0.7);
+  box-shadow: 0px 0px 4px 0.5px #{$green-light-75};
+}
+
+.Label {
+  padding-left: 8px;
+  font-size: 16px;
+  line-height: 24px;
+  user-select: none;
+  color: #{$purple-12};
+  padding-right: 4px;
+
+  .HasLabel {
+    padding-left: 4px;
+  }
+
+  .Link {
+    text-decoration: none;
+    color: #{$green-11};
+  }
+}
+
+.Label .HasOnlyLink {
+  padding-left: 4px;
+}
+
+.CheckedCheckboxRoot {
+  background-color: #{$green-11};
+}
+
+.CheckedCheckboxRoot:hover {
+  background-color: #{$green-11};
+  box-shadow: 0px 0px 4px 0.5px #{$green-light-75};
+}
+
+.DisabledCheckbox {
+  .CheckboxRoot {
+    background: #{$grey-11};
+    border: 1px solid #{$grey-9};
+    cursor: not-allowed;
+  }
+
+  .CheckedCheckboxRoot {
+    .CheckboxIndicator {
+      color: #{$grey-12};
+    }
+  }
+
+  .CheckboxRoot:hover {
+    background-color: #{$grey-11};
+    box-shadow: none;
+  }
+
+  .Label {
+    color: #{$grey-11};
+    cursor: not-allowed;
+
+    .Link {
+      color: #{$grey-9};
+    }
+  }
+}
+
+.DisabledCheckbox:hover {
+  cursor: not-allowed;
+
+  .Label:hover {
+    cursor: not-allowed;
+  }
+}
+
+.LightBg {
```

```suggestion
.LightBackground {
```

nit: let's avoid using abbreviations here for more context.

---

updated